core/src/test/resources/com/squarespace/template/plugins/platform/f-subscription-price-multiple-variants-and-multiple-pricing-options.html

```diff
@@ -0,0 +1,125 @@
+:JSON
+{
+  "structuredContent": {
+    "productType": 1,
+    "variants": [{
+      "priceMoney": {
+        "value": "10"
+      },
+      "onSale": false,
+      "pricingOptions": [{
+        "price" : "9",
```

Comment:
Not sure how much this matters for tests, but I think just for the price / priceMoney to be accurate to each other, the price is supposed to be in cents compared to the priceMoney so it would be 900 here (and other places)

Reply:
Good catch. It doesn't matter for the tests because in `pricingOptions` we only use `priceMoney` and `salePriceMoney`. But I will update it for consistency.